content/posts/visual-studio-code-setup.md

```diff
@@ -0,0 +1,142 @@
+
+### Tailwind CSS intelliSence
+
+![Tailwind CSS intelliSence in the visual studio code store](./images/tailwind-css-intellisense.png)
+
+[IntelliSence](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss) is a great feature of VSCode. Why not have it know about my Tailwind classes? Very useful for remembering the number differences for spacing and colors.
+
+### Toggle quotes
+
+![Toggle quotes in the visual studio code store](./images/toggle-quotes.png)
+
+This [extension](https://marketplace.visualstudio.com/items?itemName=BriteSnow.vscode-toggle-quotes) seems like it would be native to VSCode along with all the other text transformation tools. I can easily switch between single quotes, double quotes, and back ticks. I even like the default keyboard shortcut `control + ‘`.
+
+### Vetur
+
+![Vetur is the perfect Vue integration extension](./images/vetur.png)
+
+[Vetur](https://marketplace.visualstudio.com/items?itemName=octref.vetur) gives you easy integration with all things Vue in VSCode. It's an essential extension for working with Vue projects. What are you waiting for, download this package right now!
+
+### Vscode-icons
+
+![Example of vscode-icons for vue, javascript, etc](./images/vscode-icons.png)
+
+I've used the [vscode-icon](https://marketplace.visualstudio.com/items?itemName=vscode-icons-team.vscode-icons) set since I started using Visual Studio Code. I'm so use to them now, I don't want to switch.
+
+### Vue vscode snippets
+
+![Vue vscode snippets is the best collection of snippets for the Vue.js language](./images/vue-vscode-snippets.png)
+
+The [best snippet](https://marketplace.visualstudio.com/items?itemName=sdras.vue-vscode-snippets) package for Vue.js. I sometimes forget the format of certain features because this package has so many good snippets that I've forgot the right syntax from memory 😅
+
+### Quick and simple text selection
+
+![Quick and simple text selection in the visual studio code store](./images/quick-and-simple-text-selection.png)
+
+I've used this [extension](https://marketplace.visualstudio.com/items?itemName=dbankier.vscode-quick-select) so much I thought it baked into VSCode! It makes selecting text within characters—think `''`, `""`, `[]`, and `()`—a keyboard shortcut away. When I have to do a lot of irregular find and replace tasks, this extension haves my life.
+
+### Text Marker
+
+![Text marker extension in the visual studio code store](./images/text-marker.png)
+
+Sometimes I need to highlight a piece of code to copy, refactor, delete, etc. This [extension](https://marketplace.visualstudio.com/items?itemName=ryu1kn.text-marker) does the trick! I try to use it sparingly and clear all highlights when I'm done.
+
+## Font
+
+![Victor mono landing page](./images/font-victor-mono.png)
+
+[Victor mono](https://rubjo.github.io/victor-mono/) was my most used font in 2019. The large x-height makes it easy for me to read. Along with the ligatures and the cursive italic variant.
```

Comment:
"Along with the ligatures..." is a sentence fragment. if those things also make it easier to read, maybe "Along with the ligatures and the cursive italic variant, the large x-height makes it easy for me to read."